include/revng/Model/LoadModelPass.h

```diff
@@ -22,6 +24,8 @@ class ModelWrapper {
   ModelWrapper(TupleTree<model::Binary> &&TheBinary) :
     TheBinary(std::move(TheBinary)) {}
 
+  ModelWrapper() = default;
```

Comment:
Why?

Reply:
if it is default constructed the optional is empty and it will try to load the model from the module itself.